pkg/pquerier/pqueriertest/dataframe_query_integration_test.go

```diff
@@ -856,3 +856,106 @@ func (suite *testSelectDataframeSuite) TestQueryNonExistingMetric() {
 		}
 	}
 }
+
+func (suite *testSelectDataframeSuite) TestSparseStringAndNumericColumnsDataframe() {
+	adapter, err := tsdb.NewV3ioAdapter(suite.v3ioConfig, nil, nil)
+	suite.NoError(err, "failed to create v3io adapter")
+
+	metricCpu := "cpu"
+	metricLog := "log"
+	labels := utils.LabelsFromStringList("os", "linux")
+	labelsWithNameLog := append(labels, utils.LabelsFromStringList("__name__", metricLog)...)
+
+	expectedTimeColumn := []int64{
+		suite.basicQueryTime,
+		suite.basicQueryTime + tsdbtest.MinuteInMillis,
+		suite.basicQueryTime + 2*tsdbtest.MinuteInMillis,
+		suite.basicQueryTime + 3*tsdbtest.MinuteInMillis,
+		suite.basicQueryTime + 4*tsdbtest.MinuteInMillis}
+
+	expectedTimeColumnLog := []int64{
```

Comment:
It's not expected. it's just the times you ingest data

Reply:
ok